### PR TITLE
Pass BEM as argument to command

### PR DESCRIPTION
**Pull request by qfox**

### CODE REVIEW

**Comment by qfox** [comment]
@SevInf Is it wrong?

File: lib/coa.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 'use strict';
 
 var PATH = require('path');
+var BEM = require('../../bem');
```
Comment (arikon) [inline lib/coa.js:4]: It is wrong, because `bem` module could be anywhere in the `node_modules` "chain" (see `node` docs on `require()`)

Comment (arikon) [inline lib/coa.js:4]: We could simply use `require('bem')`. But.. whats the point passing `bem` to command functions?

Comment (qfox) [inline lib/coa.js:4]: Because if we need to have one `BEM` object with api we should pass it through `coa` to commands.
Anyway, it's failed PR.

**Pull request closed without merging**